mysql-test/suite/rpl/t/rpl_seconds_behind_master_spike.test

```diff
@@ -46,33 +42,38 @@ SET @@global.debug_dbug="+d,negate_clock_diff_with_master";
 --echo # Write events to ensure slave will be consistent with master
 create table t1 (a int);
 insert into t1 values (1);
---let $t_master_events_logged= `SELECT UNIX_TIMESTAMP()`
+--sync_slave_with_master
+let $wait_condition= SELECT STATE LIKE '%waiting for more updates'
+  FROM information_schema.PROCESSLIST WHERE COMMAND = 'Slave_SQL';
+--source include/wait_condition.inc
```

Comment:
For your awareness, @bnestere, this is a noteworthy addition since your review.

I queued the PR for merging yesterday, but the modified test sporadically [failed](https://buildbot.mariadb.org/#/builders/369/builds/32039) *in serial mode*.
I suspect it’s the same “up-to-date” assumption I had that led to #4155.